qiskit_aer/__init__.py

```diff
@@ -56,6 +56,10 @@
 # Because of this issue, we need to make sure that Numpy's OpenMP library is initialized
 # before loading our simulators, so we force it using this ugly trick
 import platform
+import sys
+import warnings
```

Comment:
Given the above comment, we probably want to keep it in close proximity to the weirdo OpenMP initialisation (if it's still even needed) - maybe move these imports below that or move the comment to immediately above the `if` rather than the import?

Reply:
I had the import below originally, pylint didn't like that so I updated it in https://github.com/Qiskit/qiskit-aer/pull/1671/commits/f1dc6b70f202dccb567ad65fed1c0ce161e981b1. I'll move the comment above the `if`. As for whether it's still needed or not I have no idea, it dates back to the very beginning of the project (and predates my involvement).

Reply:
ugh pylint lol